content/docs/next-release-v2/architecture.md

```diff
@@ -54,4 +54,64 @@ Benefits:
 
 Downsides:
 
-* An extra layer of marshaling/unmarshaling the data.
\ No newline at end of file
+* An extra layer of marshaling/unmarshaling the data.
+
+## Jaeger binary components
+
+The Jaeger binary consists of several components, some of which are derived from OpenTelemetry and some are internal. The following diagram has the major components.
+
+![Architecture](/img/architecture-v2-binary.png)
+
+Aside from these components there are sevral other components from OpenTelemetry you can use in the config of the Jaeger binary. Here is the full list of components:
+
+### Jaeger Components
```

Comment:
I am not sure we should be linking to the source code for these. We already have some documentation on the Configuration page https://deploy-preview-783--romantic-neumann-1959d7.netlify.app/docs/next-release-v2/configuration/

Reply:
I still think it's good for users to get the context of the binary and where the sources come from.